python_tests/pytest_utils.py

```diff
@@ -68,3 +71,25 @@ def make_tensor_like(a):
     return torch.testing.make_tensor(
         a.shape, device=a.device, dtype=a.dtype, requires_grad=a.requires_grad
     )
+
+
+def make_number(dtype: torch.dtype):
+    """Returns a random number with desired dtype
+
+    Args:
+        dtype (torch.dtype): Desired dtype for number.
+
+    Returns:
+        (Scalar): The scalar number with specified dtype.
+    """
+    return make_tensor([1], device="cpu", dtype=dtype).item()
+
+
+def find_nonmatching_dtype(dtype: torch.dtype):
```

Comment:
Is this still used? It looks like its one use is commented out.

Reply:
I fixed the error check.